charmstoretesting/server.go

```diff
@@ -24,12 +24,29 @@ import (
 	"gopkg.in/juju/charmstore.v4/params"
 )
 
+const (
+	// AuthUsername is the default authentication user name used by the testing
+	// charm store server if none is provided.
+	AuthUsername = "charmstore-testing-user"
+
+	// AuthPassword is the default authentication password used by the testing
+	// charm store server if none is provided.
+	AuthPassword = "charmstore-testing-password"
+)
+
 // OpenServer instantiates a new charm store server instance.
```

Comment:
// If params.AuthUsername or params.AuthPassword are empty,
// AuthUsername and AuthPassword will be used.

?